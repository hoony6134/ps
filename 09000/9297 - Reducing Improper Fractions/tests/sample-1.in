4
301 100
89 39
50 25
25 50
