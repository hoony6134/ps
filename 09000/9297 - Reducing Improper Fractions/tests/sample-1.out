Case 1: 3 1/100
Case 2: 2 11/39
Case 3: 2
Case 4: 25/50
